third_party/forked/gonum/graph/README.md

```diff
@@ -0,0 +1 @@
+Forked from gonum/graph@50b27dea7ebbfb052dfaf91681afc6fde28d8796 to support memory-use improvements to the simple graph
```

Comment:
Absolutely, I just don't want to block on the upstream PRs being accepted and vendorable prior to code freeze